docker/Dockerfile

```
@@ -67,8 +67,15 @@ RUN echo ". ~/motd.sh" >> $HOME/.bashrc
 
 # Bring Usuba
 RUN git clone https://github.com/DadaIsCrazy/usuba.git && \
-    cd usuba && ./configure.pl && opam config exec -- make
+    git clone https://github.com/usubalang/examples.git && \
+    git clone https://github.com/usubalang/benchmarks.git
+
+# Add new layer after cloning is finished
+ RUN cp -r ./examples/samples ./usuba/ && \
+    cp -r ./benchmarks/bench ./usuba/ && \
+ cd usuba && opam config exec -- make
```

`opam config exec` is deprecated:

```suggestion
 cd usuba && opam exec -- make
```

---

fixed in 417b9df